changelogs/unreleased/2627-phuong-hoang

```diff
@@ -0,0 +1 @@
+Proposes a solution that allows user to specify a backup order for resources of specific resource type.
```

Comment:
You can delete this file - it's not necessary for design documents, only for user-facing changes.

Reply:
Done.

Reply:
The master build failed ("Changelog check") without this file.  So I will have to put it back.

Reply:
If you look, the changelog check isn't required. It's just a check for us. I've also added the changelog-not-required label to this PR.